bittensor/_neuron/text/core_server/nucleus_impl.py

```diff
@@ -205,6 +205,15 @@ def remapping_token(self, token_batch, std_tokenizer=None, return_offsets_mappin
             std_tokenizer = self.std_tokenizer
 
         text_batch = std_tokenizer.batch_decode(token_batch)  # decode tokens to original text
+
+        # displaying a preview of the inference data
+        print(('Inference Data: ' + str(text_batch)[2:65]) + '......' + (str(text_batch)[-65:-2]))
+        # saving the last inference data query to a file (inference_data_last.txt in core_server/)
+        f = open('inference_data_last.txt', 'w')
+        f.write(str(text_batch)[2:-2])
+        f.write('\n')
+        f.close()
```

Comment:
We really shouldn't be writing to a file on every request. 

We should also be catching any IO errors that occur during this as otherwise it will disrupt normal function.